R/zchunk_LB132.ag_an_For_Prices_USA_C_2005.R

```diff
@@ -41,9 +42,19 @@ module_aglu_LB132.ag_an_For_Prices_USA_C_2005 <- function(command, ...) {
     FAO_USA_ag_an_P_USDt_PRICESTAT <- get_data(all_data, "aglu/FAO/FAO_USA_ag_an_P_USDt_PRICESTAT")
     FAO_USA_For_Exp_t_USD_FORESTAT <- get_data(all_data, "aglu/FAO/FAO_USA_For_Exp_t_USD_FORESTAT")
     USDA_Alfalfa_prices_USDt <- get_data(all_data, "aglu/USDA_Alfalfa_prices_USDt")
-    L100.FAO_ag_Prod_t <- get_data(all_data, "L100.FAO_ag_Prod_t")
+    FAO_ag_Prod_t_PRODSTAT <- get_data(all_data, "aglu/FAO/FAO_ag_Prod_t_PRODSTAT")
     FAO_USA_an_Prod_t_PRODSTAT <- get_data(all_data, "aglu/FAO/FAO_USA_an_Prod_t_PRODSTAT")
 
+    # Since FAO_USA_ag_an_P_USDt_PRICESTAT is in nominal years we will need a table of deflators
+    # to nomalize each to constant 1975$
+    tibble(year = aglu.MODEL_PRICE_YEARS) %>%
+      group_by(year) %>%
+      summarize(deflator = gdp_deflator(1975, year)) ->
+      conv_Price_DollarYear
+    # hard coded changes to gdp deflator for some reason:
```

Comment:
Is there a reason?

Reply:
If there isn't a good one, can we put this in a "OLD_DATA_SYSTEM_BEHAVIOR" block?

Reply:
I don't know.  This was @rynacui work originally I believe.  Maybe she has some recollection.

Reply:
This involves my work in two separate processes: DSR and core model proposal 243

In core model proposal 243, I followed the old system coding style, and manually added a deflator to 2011$ from BEA, which wasn't available in the old system. 

In DSR, I didn't incorporate the changes in the core model proposal - wasn't approved at that moment.

Anyway the short answer is we can use the deflator in the data system now, OLD_DATA_SYSTEM_BEHAVIOR would work fine.

Reply:
Thanks!  I'll make that change and the errors Ben noted and hopefully this will be wrapped up.